tests/samples/test-file.ps1

```diff
@@ -55,15 +55,10 @@ function Get-MemberSignature {
     }
 }
 
-foo "$(x).exe"
-
 function echo([string]$text) {
     write-host $text
 }
 
-$file = join-path $env:SystemDrive "$([System.io.path]::GetRandomFileName()).ps1"
-$ScriptBlock | Out-File $file -Force
-
```

Comment:
Please, revert. These are changes I added recently, did you have problems with merge from dev?